conf/mozilla-autoconfig.xml

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0"?>
 <clientConfig version="1.1">
     <emailProvider id="PRIMARY_HOSTNAME">
+      <!-- MX only -->
       <domain>PRIMARY_HOSTNAME</domain>
```

Comment:
```suggestion
      <domain purpose="mx">PRIMARY_HOSTNAME</domain>
```

https://benbucksch.github.io/autoconfig-spec/draft-ietf-mailmaint-autoconfig.html#name-domain

Thank you for the changes!

With this change applied, it's OK.